src/components/NcButton/NcButton.vue

```diff
@@ -197,6 +197,41 @@ button {
 </style>
 ```
 
+### Alignment
+Sometimes it is required to change the icon alignment on the button, like for switching between pages as in following example:
+
+```vue
+<template>
+	<div>
+		<div style="display: flex; gap: 12px;">
+			<NcButton aria-label="Previous" type="tertiary">
+				<template #icon>
+					<IconLeft :size="20" />
+				</template>
+				Previous page
+			</NcButton>
+			<NcButton alignment="end-reverse" aria-label="Previous" type="primary">
+				<template #icon>
+					<IconRight :size="20" />
+				</template>
+				Next page
+			</NcButton>
+		</div>
+	</div>
```

Comment:
(More examples)

```suggestion
	<div style="display: flex; flex-direction: column; gap: 12px;">
		<NcButton aria-label="central (default)" type="secondary" wide>
			<template #icon>
				<IconLeft :size="20" />
			</template>
			central (default)
		</NcButton>
		<div style="display: flex; gap: 12px;">
			<div style="display: flex; flex-direction: column; gap: 12px; flex: 1">
				<NcButton alignment="start" aria-label="start" type="secondary" wide>
					<template #icon>
						<IconLeft :size="20" />
					</template>
					start
				</NcButton>
				<NcButton alignment="start-reverse" aria-label="start-reverse" type="secondary" wide>
					<template #icon>
						<IconRight :size="20" />
					</template>
					start-reverse
				</NcButton>
			</div>
			<div style="display: flex; flex-direction: column; gap: 12px; flex: 1">
				<NcButton alignment="end" aria-label="end" type="secondary" wide>
					<template #icon>
						<IconLeft :size="20" />
					</template>
					end
				</NcButton>
				<NcButton alignment="end-reverse" aria-label="end-reverse" type="secondary" wide>
					<template #icon>
						<IconRight :size="20" />
					</template>
					end-reverse
				</NcButton>
			</div>
		</div>
	</div>
```

![image](https://github.com/nextcloud/nextcloud-vue/assets/25978914/0d1d92dd-171d-4c16-ba4a-0fd9ec5d89fd)

Reply:
I think the "start-reverse" and "end" layouts should not exist though, they look off and should not be used or recommended.

Reply:
@jancborchardt start-reverse and end are the values for which this was feature is introduced (table headers sort icon should be on the opposite site of the alignment)

Reply:
But as discussed with Jan, we should not promote those stypes for anything but table headers.

Reply:
> @jancborchardt start-reverse and end are the values for which this was feature is introduced (table headers sort icon should be on the opposite site of the alignment)

Do you have `end`, not `end-reversed` button in file picker table?